typescript/raven-tests.ts

```diff
@@ -24,7 +24,11 @@ var options: Raven.RavenOptions = {
         xhr: false,
         console: false,
         dom: true,
-        location: false
+        location: false,
+        sentry: true
```

Comment:
`sentry` is sentry event breadcrumbs?

Reply:
Yes, it has been added here https://github.com/getsentry/raven-js/pull/1099